Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -253,30 +253,43 @@ function vtkWidgetManager(publicAPI, model) {
   // API public
   // --------------------------------------------------------------------------
 
-  async function updateSelection({ position }) {
-    model._selectionInProgress = true;
+  model._updateSelectionCallID = null;
```

Comment:
does not seem the right place to initialize model variable

Reply:
I moved this into a comment in DEFAULT_VALUES.

Reply:
sounds good, but this variable is never read. are you sure there is no typo ?

Reply:
Ah, that is a typo. I've updated it with the right name.